sparse/__init__.py

```diff
@@ -45,13 +45,7 @@ def get_backend_module():
 
 
 def __getattr__(attr):
-    if attr == "numba_backend":
-        import sparse.numba_backend as backend_module
-
-        return backend_module
-    if attr == "finch_backend":
-        import sparse.finch_backend as backend_module
-
-        return backend_module
+    if attr == "numba_backend" or attr == "finch_backend":
+        raise AttributeError
```

Comment:
Let's merge https://github.com/pydata/sparse/pull/695 first. Then these changes can go away.

Reply:
Done in https://github.com/pydata/sparse/pull/696/commits/a2e84d5320bdae456ea38c5ef71c11eb5b366226.